wpuf-functions.php

```
@@ -5990,3 +5984,155 @@ function wpuf_render_login_layout_field( $args ) {
+    $post = get_post( $post_id );
+
+    if ( ! $post || is_wp_error( $post ) ) {
+        return new WP_Error(
+            'wpuf_post_not_found',
+            __( 'Post not found.', 'wp-user-frontend' )
+        );
+    }
+
+    // Get current user and post author
+    $current_user_id = get_current_user_id();
+    $post_author_id  = (int) $post->post_author;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$post_author_id'. (undefined)

(UnusedLocalVariable)